src/nupic/algorithms/Cells4.hpp

```diff
@@ -406,8 +407,8 @@ namespace nupic {
                Real permInc =.1,
                Real globalDecay =0,
                bool doPooling =false,
-               int seed =-1,
-               bool doItAll =false,
+               int seed =42,
+               bool initFromCpp =true,
```

Comment:
Actually, here I think it's more useful to default to `true`, as 
* when called from a "wrapper" in Python, the param is set to False. 
* should this class be used from c++ directly, the `true` is mandatory (and so reasonable default)

Also, overwriting just some default params is a beauty in python, and sucks in c++, thus my motivation for the defaults here